lib/configurators/aws/aws_test.go

```diff
@@ -1317,6 +1318,13 @@ type SSMMock struct {
 func (m *SSMMock) CreateDocumentWithContext(ctx aws.Context, input *ssm.CreateDocumentInput, opts ...request.Option) (*ssm.CreateDocumentOutput, error) {
 
 	m.t.Helper()
+
+	// UUID's are unpredictable, so we remove them from the content
+	uuidRegex := regexp.MustCompile(`[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}`)
+	replacedExpected := uuidRegex.ReplaceAllString(*m.expectedInput.Content, "")
+	m.expectedInput.Content = &replacedExpected
+	replacedInput := uuidRegex.ReplaceAllString(*input.Content, "")
+	input.Content = &replacedInput
 	require.Equal(m.t, m.expectedInput, input)
```

Comment:
Maybe?

```suggestion
	// Diff content first for a nicer error message.
	require.Empty(m.t,
		cmp.Diff(m.expectedInput.Content, input.Content),
		"Document content diff (-want +got)")
	require.Empty(m.t,
		cmp.Diff(m.expectedInput, input, cmpopts.IgnoreFields(ssm.CreateDocumentInput{}, "Content")),
		"Document diff (-want +got)")
```

cmpopts is github.com/google/go-cmp/cmp/cmpopts

Reply:
Updated, thank you